tiles/src/main/java/com/protomaps/basemap/layers/Places.java

```diff
@@ -158,7 +187,17 @@ public void processFeature(SourceFeature sf, FeatureCollector features) {
         }
       }
 
-      feat.setAttr("population_rank", population_rank);
+      feat.setAttr("pmap:population_rank", population_rank);
+
+      if (population > 0) {
+        feat.setAttr("population", population);
+
+        // we set the sort keys so the label grid can be sorted predictably (bonus: tile features also sorted)
+        feat.setPointLabelGridPixelSize(12, 128);
+        //feat.setSortKey(getSortKey("pmap:min_zoom",  "pmap:population_rank", "population", "name"));
```

Comment:
@bdon 2nd place to re-enable the sort key stuff once Java type conversion problems are fixed